packages/headless/src/features/insight-search/insight-search-actions.ts

```diff
@@ -180,7 +183,14 @@ export const fetchMoreResults = createAsyncThunk<
     ...config,
   });
 
-  const request = await buildInsightFetchMoreResultsRequest(state);
+  const eventDescription = {
```

Comment:
Why isn't this using `buildEventDescription` ?

Reply:
updated.